assets/js/src/newsletters/types/re_engagement/re_engagement.tsx

```diff
@@ -1,21 +1,34 @@
-import React from 'react';
+import React, { useState } from 'react';
 
 import MailPoet from 'mailpoet';
 import Background from 'common/background/background';
 import { Grid } from 'common/grid';
 import Button from 'common/button/button';
 import ListingHeadingStepsRoute from '../../listings/heading_steps_route';
-import Heading from '../../../common/typography/heading/heading';
+import { Scheduling } from './scheduling';
+
+interface ReengagementWindow extends Window {
+  settings: {
+    deactivate_subscriber_after_inactive_days: string;
+  }
+}
+
+declare let window: ReengagementWindow;
 
 export function NewsletterTypeReEngagement(): JSX.Element {
+  const [options] = useState({
+    afterTimeNumber: window.settings.deactivate_subscriber_after_inactive_days,
```

Comment:
I think we wanted to avoid adding new usages of the global window within components. What about adding the settings object to the MailPoet module? 

I think we already have a type definition for settings https://github.com/mailpoet/mailpoet/blob/2860db41e588400d8754dc6273fa7839b4f648d0/assets/js/src/settings/store/types.ts#L1-L137